lib/parity/environment.rb

```diff
@@ -57,9 +57,9 @@ def deploy_to_heroku
     end
 
     def restore
-      if production?
+      if production? && additional_restore_arguments["--force"].nil?
```

Comment:
Whoaaaa I've never seen a string check thing like this. At first glance I thought `additional_restore_arguments` was a hash. TIL.

Thoughts on using regex like `additional_restore_arguments =~ /--force/` to clarify?